packages/core/components/Modal/CopyFileManifest/CopyFileManifest.module.css

```diff
@@ -63,10 +82,24 @@
     z-index: 3;
 }
 
+.file-table th:last-child,
+.file-table td:last-child {
+    width: 10em;
+    min-width: 10em;
+    max-width: 10em;
+}
+
+.file-table th:first-child,
+.file-table td:first-child {
+    width: calc(100% - 10em);
```

Comment:
Since `10em` is used multiple times where it's important to keep it identical, I recommend introducing it as a CSS variable.

Also, 10em might be a tad too large? Just judging by the whitespace in your example movie.

Reply:
I'd add to this that the right padding on .body-container makes it ever so slightly off-center which I think makes the left margin look smaller than it is

Reply:
ah yeah that padding was originally for the scroll bar but is no longer needed.

Reply:
resolved in [82c5463](https://github.com/AllenInstitute/biofile-finder/pull/421/commits/82c546325862e26756bf900003a417604881a9f5)